src/Swarm/Game/State.hs

```diff
@@ -736,6 +744,7 @@ initGameState = do
   return $
     GameState
       { _creativeMode = False
+      , _gameStep = RobotStep LT
```

Comment:
@byorgey please give it a try - I set it to the robot step by default for testing. :slightly_smiling_face:

Reply:
Hmm, is that what makes it so terribly slow?  I mean when you just let it run without being paused, for me it runs at about 0.5 ticks per second...

Reply:
Yes, maybe single stepping focused robot should not be allowed outside of paused mode. 😄

I did not add a switch to turn it on/off yet.